Fix project page styling

yieb opened the pull request.

yieb committed:
Fix mixed up footer logos

yieb committed:
Fix styling with Projects page

heylookitsme commented:
![image](https://user-images.githubusercontent.com/45301066/183258087-b996fc01-f5e0-4fc6-93aa-13d7ff074c41.png)

Commit:
Changed image resizing for mobile

- Added projectPicture class to apply styling to pictures on project
  pages
- Patched projectImage and projectInfoContainer mobile briefly,
      * used same strategy to depict picture as with the current
	project/featured project cards because it works, on
	projectPicture class
      * flexChild is the class used for the image and info. should
	consider renaming to something more descriptive.
- removed min-width:989px from ProjectsPage because the margin was too
  big (and was not adherent with figma representation of project page
  https://www.figma.com/file/PfdQ5LLTKj36HgTnkdU0yU/Website-Wireframe?node-id=1051%3A2355)
